servers/rendering/rendering_device.h

```diff
@@ -245,13 +255,34 @@ class RenderingDevice : public RenderingDeviceCommons {
 			r.layer_count = layers;
 			return r;
 		}
+
+		TextureFormat texture_format() const {
```

Comment:
In Godot the naming convention in cases like this involves adding a `get_` prefix. However, since there are already other cases of this other style in the code, I guess a future PR (by maybe @akien-mga or @AThousandShips) can do a uniformity pass.